database/migrations/2020_02_21_233331_add_experience_education_table.php

```diff
@@ -26,6 +26,7 @@ public function up()
             $table->boolean('has_blockcert')->nullable();
             $table->integer('experienceable_id')->unsigned()->index();
             $table->string('experienceable_type');
+            $table->boolean('is_education_requirement')->default(true);
```

Comment:
Hmm. Good question, but I think we should leave it default(false) for consistency, and leave it up to the user. We can always suggest it in the UI if we want to.